example/cmd/device-simple/res/devices/simple-device.json.example

```diff
@@ -0,0 +1,16 @@
+{
+  "Name": "Simple-Device01",
```

Comment:
JSON element names should start lower-case, ie "name", "profileName", "autoEvents", "onChange" etc

Reply:
updated.